contentcuration/contentcuration/frontend/shared/views/channel/ChannelModal.vue

```diff
@@ -154,20 +165,24 @@
       },
       currentTab: {
         get() {
-          const sharing = this.$route.query.sharing;
-          // On load, sharing counts as string, so just process as if a string
-          return sharing && String(sharing) === 'true' ? 'share' : 'edit';
+          const tab = this.tab;
+          return tab;
         },
         set(value) {
           // Only navigate if we're changing locations
-          if (value !== this.currentTab) {
-            this.$router.replace({
-              ...this.$route,
-              query: {
-                ...this.$route.query,
-                sharing: value === 'share',
-              },
-            });
+          if (value !== this.tab) {
+            this.$router
+              .replace({
+                ...this.$route,
+                query: {
+                  ...this.$route.query,
+                },
+                params: {
+                  ...this.$route.params,
+                  tab: value,
+                },
+              })
+              .catch(() => {});
```

Comment:
I don't remember off the top of my head, but I know there was a purpose, as I remember researching it. I will see if I can look it up again and I will follow up with you!

Reply:
I am afraid that this might result in losing information or swallowing errors that need to be handled. It's okay if there is a reason - it would be nice to add the information to the comment.